frontend/src/components/assessments/student-experience/question-elements/MultiSelect.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Checkbox, CheckboxGroup, VStack } from "@chakra-ui/react";
+
+import useAnswerState from "./useAnswerState";
+
+interface MultiSelectProps {
+  answerElementIndex: number;
+  options: string[];
+}
+
+const MultiSelect = ({
+  answerElementIndex,
+  options,
+}: MultiSelectProps): React.ReactElement => {
+  const { currentAnswer, updateAnswer } = useAnswerState(answerElementIndex);
+
+  return (
+    <CheckboxGroup
+      onChange={(e) => updateAnswer(e as number[])}
```

Comment:
argh I hate that a cast is needed here :/

Reply:
RIGHT!